java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -219,7 +219,11 @@ public enum OrcConf {
       "orc.proleptic.gregorian.default", false,
       "This value controls whether pre-ORC 27 files are using the hybrid or proleptic\n" +
       "calendar. Only Hive 3.1 and the C++ library wrote using the proleptic, so hybrid\n" +
-      "is the default.")
+      "is the default."),
+  ROW_BATCH_SIZE("orc.row.batch.size", "orc.row.batch.size", 1024,
+          "The number of rows to include in a orc vectorized reader batch. " +
+                  "The number should be carefully chosen to minimize overhead " +
+                  "and avoid OOMs in reading data.")
```

Comment:
The following will be enough. I also change `number` to `value` in the second sentence.
```java
  ROW_BATCH_SIZE("orc.row.batch.size", "orc.row.batch.size", 1024,
      "The number of rows to include in a orc vectorized reader batch. " +
      "The value should be carefully chosen to minimize overhead and avoid OOMs in reading data.")
```

Reply:
@cxzl25 . Here I changed your indentation too. I understand the indentation is heterogenous in this file, but please follow my example.

Reply:
Sorry I didn't notice the indentation issue here, I was breaking lines because it was over 100 characters. After changing the indentation, it is now 98 characters, thanks.